File: 2000 98 & ME/QuickStart/howto/samples/Services/MessageQueue/sendobject/vb/sendobject.vb
Imports System
Imports System.Messaging
	
Public Class SendObject

    Public Shared Sub Main() 

        Dim mqPath As String = ".\MQSendSample"

        if(Not MessageQueue.Exists(mqPath)) then
            MessageQueue.Create(mqPath)
        end if
		
        Dim mq As MessageQueue = new MessageQueue(mqPath)

        Dim customer As Customer = new Customer()
        customer.LastName = "Copernicus"
        customer.FirstName = "Nicolaus"		
        mq.Send(customer)     

    End Sub

End Class

File: 2000 98 & ME/QuickStart/howto/samples/Services/MessageQueue/sendobject/sendobject.cs
using System;
using System.Messaging;
	
public class SendObject
{
        public static void Main(String[] args) 
	{
		string mqPath = ".\\MQSendSample";

		if(!MessageQueue.Exists(mqPath))
		{
			MessageQueue.Create(mqPath);
		}
		
		MessageQueue mq = new MessageQueue(mqPath);

		Customer customer = new Customer();
		customer.LastName = "Copernicus";
		customer.FirstName = "Nicolaus";		
		mq.Send(customer);	
        }
}